ci/pipelines/tasks/prepare-env.yml

```diff
@@ -0,0 +1,27 @@
+inputs:
+- name: source
+
+outputs:
+- name: source
+
+platform: linux
+params:
+    IS_CI: true
+run:
+    args:
+    - -exc
+    - |
+        export GRADLE_USER_HOME=$(pwd)/.gradle
+
+        cd source
+        chmod u+x gradlew
+        echo taplinxKey=((apiary-android-secrets.taplinxKey)) >> local.properties
+        echo taplinxOfflineKey=((apiary-android-secrets.taplinxOfflineKey)) >> local.properties
+        echo sentryDsn=((apiary-android-secrets.sentryDsn)) >> local.properties
+        
+        gem install bundler:2.3.1
+        bundle config set --local path vendor
+        bundle install --quiet
```

Comment:
Can you cache any of this?

Reply:
Like cache between builds?  The `bundle install` output is used for all later tasks where it's needed

Reply:
Yes, caching between builds.